common/src/main/java/com/microsoft/identity/common/internal/ui/embeddedwebview/EmbeddedWebViewAuthorizationStrategy.java

```diff
@@ -71,25 +75,24 @@ public GenericAuthorizationResult requestAuthorization(final GenericAuthorizatio
      * @throws UnsupportedEncodingException thrown when the Character Encoding is not supported
      * @throws ClientException throw when error happens during the authorization
      */
-    public EmbeddedWebViewAuthorizationStrategy(final GenericWebViewClient webViewClient)
+    public EmbeddedWebViewAuthorizationStrategy(final GenericWebViewClient webViewClient, final WebView webView)
             throws UnsupportedEncodingException, ClientException {
         //TODO validate auth request in OAuth2Strategy.
-        createWebView(webViewClient);
+        createWebView(webViewClient, webView);
         mStartUrl = webViewClient.getRequest().getAuthorizationStartUrl();
     }
 
     /**
      * Set up the web view configurations.
-     * @param activity  AuthenticationActivity
      * @param webViewClient AzureActiveDirectoryWebViewClient
      */
-    private void createWebView(final GenericWebViewClient webViewClient) {
-        final Activity activity = webViewClient.getActivity();
+    private void createWebView(final GenericWebViewClient webViewClient, final WebView webView) {
```

Comment:
Given that this method accepts a `WebView` as a parameter, the name `createWebView()` seems a bit of a misnomer.... maybe.... `setUpWebView()` or `initializeWebView()`? Open to other thoughts :-)